compiler/koin-ksp-compiler/src/jvmMain/kotlin/org/koin/compiler/generator/KoinGenerator.kt

```diff
@@ -43,19 +43,20 @@ class KoinGenerator(
 
         if (defaultModule.definitions.isNotEmpty()) {
             logger.logging("generate default module ...")
-            val defaultModuleFile = codeGenerator.getDefaultFile()
+            val defaultModuleFile = codeGenerator.getFile(fileName = "Default")
```

Comment:
is it the same here?

Reply:
getDefaultFile cached default file as local field with cause issue with multiple pass and wasn't necessary. The result behavior is the same, but without local field

Reply:
ok, got it